### ISSUE TITLE
NVDA drag and drop fails in centovacast 3.0

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3449]
**Reported by lilmike on 2013-08-19 00:47**
Hi,
When I am in centovacast, a streaming radio hosting control panel, there is some drag and drop required to put songs into playlists. However, when I do the following:
1. move the mouse cursor to the navigation point and click it,
2. lock the mouse and navigation to where  I want to drop it,
3. unlock the mouse, an error occurs in centova. I have contacted them about this, and they say the way I describe doing this is exactly what a sighted person would do to drag and drop the text, so I am thinking NVDA doesn't have some feature necessary for the javascript to recognize that it dragged an item onto it. IF you need a test account to try this on I can give you one on request.

Thanks,
-Michael.

**Comment by leonardder** [comment]
Hmm, I've been trying to drag and drop icons in the task bar recently, and it seems to fail regularly. Though that's not related to this particular application, it is certainly related to drag and drop behaviour.

CC @lilmike: Is this still of interest for you?

**Comment by Adriani90** [comment]
@leonardder in the taskbar on Windows 10 1809 I encounter the same problem. NVDA opens the application instead of dragging and dropping it to the desired location.

**Comment by Adriani90** [comment]
We didn't get any updates from the initial author since many years. I am closing as abandoned. Feel free to comment with an update on this issue and we can reopen.
